flake.nix

```diff
@@ -3,180 +3,25 @@
-            system = "x86_64-linux";
-            modules = [
-              dummy
-              fake-hardware
-              self.nixosModules.mixins-telegraf
-            ];
-          };
-          example-mixins-terminfo = nixpkgs.lib.nixosSystem {
-            system = "x86_64-linux";
-            modules = [
-              dummy
-              fake-hardware
-              self.nixosModules.mixins-terminfo
-            ];
-          };
-          example-mixins-trusted-nix-caches = nixpkgs.lib.nixosSystem {
-            system = "x86_64-linux";
-            modules = [
-              dummy
-              fake-hardware
-              self.nixosModules.mixins-trusted-nix-caches
-            ];
-          };
-          example-mixins-nginx = nixpkgs.lib.nixosSystem {
-            system = "x86_64-linux";
-            modules = [
-              dummy
-              fake-hardware
-              self.nixosModules.mixins-nginx
-            ];
-          };
-
-          # Roles
-          example-roles-github-actions-runner = nixpkgs.lib.nixosSystem {
-            system = "x86_64-linux";
-            modules = [
-              self.nixosModules.roles-github-actions-runner
-              dummy
-              fake-hardware
-              {
-                roles.github-actions-runner.cachix.cacheName = "cache-name";
-                roles.github-actions-runner.cachix.tokenFile = "/run/cachix-token-file";
-                roles.github-actions-runner.tokenFile = "/run/gha-token-file";
-                roles.github-actions-runner.url = "https://fixup";
-              }
-            ];
-          };
-          example-roles-github-actions-runner-github-app = nixpkgs.lib.nixosSystem {
-            system = "x86_64-linux";
-            modules = [
-              self.nixosModules.roles-github-actions-runner
-              dummy
-              fake-hardware
-              {
-                roles.github-actions-runner.cachix.cacheName = "cache-name";
-                roles.github-actions-runner.cachix.tokenFile = "/run/cachix-token-file";
-                roles.github-actions-runner.githubApp = {
-                  id = "1234";
-                  login = "foo";
-                  privateKeyFile = "/run/gha-token-file";
-                };
-                roles.github-actions-runner.url = "https://fixup";
-              }
-            ];
-          };
-          example-roles-nix-remote-builder = nixpkgs.lib.nixosSystem {
-            system = "x86_64-linux";
-            modules = [
-              self.nixosModules.roles-nix-remote-builder
-              dummy
-              fake-hardware
-              {
-                roles.nix-remote-builder.schedulerPublicKeys = [
-                  "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOuiDoBOxgyer8vGcfAIbE6TC4n4jo8lhG9l01iJ0bZz zimbatm@no1"
-                ];
-              }
-            ];
-          };
-        };
+      imports = [
+        ./modules/flake-parts/all-modules.nix
```

Comment:
Let's move this out of the ./modules folder to make the DAG non-cyclic.

Reply:
Yeah, it's more for the mental model, so the user doesn't have to dig into the code